cmd/admin_user_create.go

```diff
@@ -191,6 +200,7 @@ func runCreateUser(c *cli.Context) error {
 		Passwd:             password,
 		MustChangePassword: mustChangePassword,
 		Visibility:         visibility,
+		FullName:           fullName,
```

Comment:
You could use `FullName: c.String("fullname")` directly without the `if IsSet` block.

Just one-line.

Reply:
Good point, will adjust